packages/react/src/use-chat-2.ts

```diff
@@ -43,38 +43,6 @@ export type UseChatHelpers2<
     InferUIDataParts<DATA_PART_SCHEMAS>
   >[];
 
-  /**
-   * Append a user message to the chat list. This triggers the API call to fetch
-   * the assistant's response.
-   *
-   * @param message The message to append
-   * @param options Additional options to pass to the API call
-   */
-  append: (
-    message: CreateUIMessage<
-      MESSAGE_METADATA,
-      InferUIDataParts<DATA_PART_SCHEMAS>
-    >,
-    options?: ChatRequestOptions,
-  ) => Promise<void>;
-
-  /**
-   * Reload the last AI chat response for the given chat history. If the last
-   * message isn't from the assistant, it will request the API to generate a
-   * new response.
-   */
-  reload: (chatRequestOptions?: ChatRequestOptions) => Promise<void>;
-
-  /**
-   * Abort the current request immediately, keep the generated tokens if any.
-   */
-  stop: () => void;
-
-  /**
-   * Resume an ongoing chat generation stream. This does not resume an aborted generation.
-   */
-  experimental_resume: () => void;
-
```

Comment:
pushed these back to `AbstractChat` by renaming some of its methods to match these and attaching the JSDoc there (so that it can be inherited by all consumers!)